### PR TITLE
Token page

### LINKED ISSUE #593: Implement the /tokens/[tokenId] page
This page will present a single token: capture/tree picture,  tags, just like the tree page

We will re-use the tree page design for this implementation: https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=7639%3A30959

The API is ready: https://github.com/Greenstand/treetracker-query-api/blob/main/__tests__/e2e/tokens.spec.ts

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by Mohmn**
# Description

[comment]: # 'Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.'

Fixes #593

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

# How Has This Been Tested?

- [x] Cypress integration
- [ ] Cypress component tests
- [ ] Jest unit tests

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Mohmn** [comment]
@dadiorchen  theme toggling isn't working so I only attached default mode screenshots.
 check my code for setting the filter for the big map, I checked for the documentation but couldn't find it
so I intuitively took a guess here

**Comment by Mohmn** [comment]
Desktop 
Screenshot from 2022-07-07 00-41-20
Screenshot from 2022-07-07 00-39-46
Screenshot from 2022-07-07 00-39-42
Screenshot from 2022-07-07 00-39-36

**Comment by Mohmn** [comment]
mobile
Screenshot_2022-07-07-01-19-52-614_com android chrome
Screenshot_2022-07-07-01-19-49-277_com android chrome

**Comment by dadiorchen** [comment]
> 

Please open an issue about theme toggle and assign it to @RubenSmn

**Comment by RubenSmn** [comment]
@dadiorchen @Mohmn the theme toggle feature will be fixed with #654

**Comment by RubenSmn** [comment]
A temporary work around could be changing the theme value to `light` in the localstorage

**Comment by Mohmn** [comment]
> A temporary work around could be changing the theme value to `light` in the localstorage

I had already tried it but still wasn't able to change the theme

**Comment by dadiorchen** [comment]
@Mohmn I am going to merge this PR, but before that, can you merge the newest code and check if all the CI tests pass? I have fixed the integration tests, so if you update the code, the CI should pass now

File: src/pages/tokens/[tokenid].js [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+  };
+}
+
+// eslint-disable-next-line require-await
+export async function getStaticPaths() {
+  return {
+    paths: [],
+    fallback: 'blocking',
+  };
+}
```
Comment (dadiorchen) [inline src/pages/tokens/[tokenid].js:330]: @Mohmn please change this to the new pattern of getServerSideProp, I temporarily removed the `cache` feature, and plan to use it in the future when we are ready, this feature is potentially causing problems.

**Comment by dadiorchen** [comment]
@Mohmn I have solved the CI failure for you, https://github.com/Greenstand/treetracker-web-map-client/pull/664/commits/8749b3e1cb462754318dd54057af5f2d542bd53d

**Comment by dadiorchen** [comment]
Big progress, thank you!

File: src/pages/tokens/[tokenid].js (hunk 1 above)
Comment (Mohmn) [reply src/pages/tokens/[tokenid].js:330]: sorry for the delay
will update it

**Comment by dadiorchen** [comment]
@Mohmn no need to do anything on this, I have solved the problem.

**Pull request merged**